cypress/support/pages/admin/pageAdministration.js

```diff
@@ -172,19 +172,18 @@ class Management {
 		cy.get(Management.#studentTeamCheckbox).should("be.checked");
 	}
 
-	navigateToAdministration() {
+	openAdministrationInMenu() {
 		cy.get(Management.#administrationOverviewNavigationButton).click();
-		cy.url().should("include", "/administration");
 	}
 
-	navigateToStudentAdministration() {
-		cy.get(Management.#studentAdministrationNavigationButton).click();
-		cy.url().should("include", "/administration/students");
-	}
-
-	navigateToTeacherAdministration() {
-		cy.get(Management.#teacherAdministrationNavigationButton).click();
-		cy.url().should("include", "/administration/teachers");
+	navigateToUserAdministration(role) {
+		if (role === "student") {
```

Comment:
This method can be improved. Instead of multiple if-else block we can approach something else. My suggestion would be:

- Defining a variable, which holds the expected URL based on the provided role (student or teacher).
- Using a ternary operator to conditionally assign the appropriate navigation button selector based on the role.

after doing this we have two line:
```js
cy.get(<varibale which we assign in the ternary operator>).click();
cy.url().should("include", "/administration/<role>");
```